**Задача №1.** Инвестор купил акцию 6 лет назад по цене $10. Сейчас он продал ее за 50 долларов. Определить, какую доходность принесла ему эта инвестиция в процентах годовых.

Для расчета доходности, которая принесла инвестиция в процентах годовых будем рассчитывать используя следующую формулу: FV = PV * (1 + r)^n, откуда ставка накопления (дисконтирования):

$$ r = \left(\frac{FV}{PV}\right)^{\frac{1}{n}} - 1, где: $$

$FV$ - будущая стоимость ден.ед.,<br>
$PV$ - начальная стоимость ден.ед.,<br>
$n$ - количество лет (период времени), в течение которых была сделана инвестиция.

In [3]:
FV = 50
PV = 10
n = 6

In [9]:
r = ((FV/PV) ** (1/6)) - 1
print(round(r, 4))

0.3077


**Ответ:** Доходность инвестиции 30.77% годовых.

**Задача №2.** Инвестору исполнилось только что 30 лет. Он хочет выйти на пенсию в 60 лет и жить на доход с капитала. Допустим, его целевой уровень капитала к пенсии составляет $350 000. Если он ожидает ставку доходности на рынке порядка 8% годовых, то какую сумму ему надо инвестировать каждый год для достижения цели?

Для определения необходимой суммы инвестиций каждый год для достижения цели воспользуемся следующей формулой:

$$ PMT = \frac{FV \cdot r}{(1 + r)^n - 1}, где: $$

$PMT$ - платеж, необходимый для накопления в будущем определенной суммы в ден. ед.,<br> $FV$ - будущая стоимость инвестиции, <br> $r$ - ставка накопления дисконтирования в доли ед., <br> $n$ - интервал времени, период времени


In [6]:
FV = 350000
r = 0.08
n = 60-30

In [10]:
PMT = (FV * r) / (((1+ r) ** n) - 1)
print(round(PMT, 2))

3089.6


**Ответ:** Для достижения цели (уровень капитала к пенсии 350 000 $) 30 летнему инвестору необходимо инвестировать каждый год 3089.60 долларов.

**Задача №3.** Человек взял ипотечный кредит на сумму 8 млн руб., на 20 лет под 10% годовых. Погашение кредита будет происходить ежемесячными аннуитетными платежами. Определить, сколько составит общая переплата (сумма процентов) по кредиту.

Для начала определим размер ежемесячного платежа по кредиту, для этого воспользуемся следующей формулой:

$$ PMT = \frac{PV \cdot r}{1 - (1 + r)^-n }, где: $$

$PV$ - текущая стоимость, ден.ед.,<br>
$r$ - ставка накопления дисконтирования в доли ед., <br>
$n$ - интервал времени, период времени (месяцы)

In [19]:
PV = 8000000
r = (10/12/100)
n = 20*12
print(r, n)

0.008333333333333333 240


In [20]:
PMT = (PV * r) / ((1 - (1+ r) ** (-n)))
print(round(PMT, 2))

77201.73


Найдем общую сумму выплат за весь период кредита:$$ ∑PMT = PMT * n $$

In [25]:
sumPMT = PMT * n
print(round((sumPMT), 2))

18528415.59


Общая переплата по кредиту составит:


In [28]:
overpayment = sumPMT - PV
print(round((overpayment), 2))

10528415.59


**Ответ:** Общая переплата по кредиту составляет: 10 528 415,59 руб.

**Задача №4.** Известно, что безрисковая ставка на рынке составляет 1%, инфляция ожидается 6% годовых и для данного проекта премия за риск равна 4%. Пусть ставка дисконтирования определяется как сумма этих трех составляющих, тогда чему равна приведенная стоимость потоков по проекту, если в первый год ожидается 2000 долларов, во второй 5000 долларов и в третьем году проект будет продан за $10000?

Для расчета приведенной стоимости потоков по проекту воспользуемся следующей  формулой:
$$ PV =\frac{FV}{(1 + r)^n}, где: $$

$FV$ - будущая стоимость ден.ед.,<br>
$r$ - ставка накопления дисконтирования в доли ед., <br>
$n$ - количество лет (период времени)

Откуда формула приведенной стоимости потоков за три года примет следующий вид:
$$
PV = \frac{FV_1}{(1 + r)^1} + \frac{FV_2}{(1 + r)^2} + \frac{FV_3}{(1 + r)^3}
$$

Ставка дисконтирования согласно условию задачи определим как: сумму безрисковой ставки (1%), ожидаемой инфляции (6%) и премии за риск (4%). <br>
Итого: r = 11%

In [1]:
r = 11 / 100
FV1 = 2000
FV2 = 5000
FV3 = 10000

In [2]:
PV = FV1 / ((1 + r) ** 1) + FV2 / ((1 + r) ** 2) + FV3 / ((1 + r) ** 3)
print(round((PV), 2))

13171.83


**Ответ:** Приведенная стоимость потоков по проекту составит: 13 171.83 $

**Задача №5.** Что выгодней: положить деньги на депозит под 11% годовых с ежемесячной капитализацией или на депозит под 11,5% с ежегодной капитализацией процентов?

In [7]:
r1 = 11/100
r2 = 11.5/100
n = 12
t = 1

Эффективную годовую процентную ставку EIR для депозита с ежемесячной капитализацией, определим по следующей формуле:

$$ EIR = (1 + \frac{r_1} {n}) ^{n*t} - 1, где: $$

$r_1$ — ставка накопления дисконтирования в доли ед., <br>
n — количество периодов капитализации в году (месяцев),<br>
t — срок депозита в годах

In [11]:
EIR = ((1 + (r1/n)) ** (n * t)) - 1
print((round((EIR), 4) * 100))

11.57


Итого: эффективная годовая процентная ставка EIR для депозита с ежемесячной капитализацией составила: 11.57 %

**Ответ:** Таким образом исходя из выше приведенного расчета, выгоднее положить деньги на депозит под 11% годовых с ежемесячной капитализацией.

